plugins/harness-ci-cd/src/hooks/useGetExecutionsList.ts

```diff
@@ -50,8 +52,8 @@ const useGetExecutionsList = ({
         }
       });
     }
-
-    const token = getSecureHarnessKey('token');
+    const { token: apiToken } = await identityApi.getCredentials();
+    const token = apiToken || getSecureHarnessKey('token');
```

Comment:
```suggestion
    const { token: apiToken } = await identityApi.getCredentials();
    const token = getSecureHarnessKey('token') || apiToken ;
```

Reply:
We should prioritise harnessToken because the backstage token will be always present. For integration with Harness IDP harnessToken is a must and it is only present in that scenario only.